spectator-api/src/main/java/com/netflix/spectator/api/AbstractRegistry.java

```diff
@@ -51,9 +52,23 @@ public AbstractRegistry(Clock clock) {
    *     Configuration settings for the registry.
    */
   public AbstractRegistry(Clock clock, RegistryConfig config) {
+    this(clock, config, new ConcurrentHashMap<>(1024));
```

Comment:
the concurrency level is derived from the initial size with this particular constructor:
        int cap = ((initialCapacity >= (MAXIMUM_CAPACITY >>> 1)) ?
                   MAXIMUM_CAPACITY :
                   tableSizeFor(initialCapacity + (initialCapacity >>> 1) + 1));